skymp5-server/cpp/server_guest_lib/ActionListener.cpp

```diff
@@ -648,6 +649,14 @@ void ActionListener::OnHit(const RawMessageData& rawMsgData_,
     return;
   }
 
+  std::unordered_set<uint32_t> greenZones = { 0x0760AADA };
+  uint32_t aggressorCell =
+    aggressor->GetCellOrWorld().ToFormId(aggressor->GetParent()->espmFiles);
+  auto it = greenZones.find(aggressorCell);
+  if (it != greenZones.end()) {
+    return;
+  }
```

Comment:
Does it work or will it be replaced with an alternative implementation?

Reply:
Doesn't, I'll remove it asap